soapui/src/main/java/com/epam/reportportal/soapui/service/StepBasedSoapUIServiceImpl.java

```diff
@@ -118,20 +131,25 @@ public void finishTestSuite(TestSuiteRunner testSuiteContext) {
 	}
 
 	public void startTestCase(TestCase testCase, PropertyExpansionContext propertyContext) {
-		Maybe<String> id = startItem(testCase.getName(),
-				TestItemType.TEST_CASE,
-				fromStringId(testCase.getTestSuite().getPropertyValue(ID))
-		);
+		Maybe<String> id = startItem(testCase, TestItemType.TEST_CASE, fromStringId(testCase.getTestSuite().getPropertyValue(ID)));
 		testCase.setPropertyValue(ID, toStringId(id));
 	}
 
-	protected Maybe<String> startItem(String name, TestItemType type, Maybe<String> parentId) {
+	protected Maybe<String> startItem(TestCase testCase, TestItemType type, Maybe<String> parentId) {
 		if (null != launch) {
 			StartTestItemRQ rq = new StartTestItemRQ();
-			rq.setName(name);
+			rq.setName(testCase.getName());
```

Comment:
Can we move building the startRq to the separate method, so we can it test separately

Reply:
It is not necessary, because we should test the original request